src/styles/main.scss

```diff
@@ -1,3 +1,38 @@
 body {
   margin: 0;
+  height: 100vh;
```

Comment:
This parameter is redundant because we should have fixed hight of blocks
```suggestion
```

Reply:
![image](https://user-images.githubusercontent.com/55802630/155513430-eda6bc64-01be-4937-a1de-c20ca4fe420a.png)
ilia said snake must be fullscreen, in last task. I think this one also should be fullscreen